server/dataGeneration/generateCopyReviews.js

```diff
@@ -1,7 +1,7 @@
 const fs = require('fs');
 const path = require('path');
 
-const query = 'COPY public.reviews (author, date, rating, content, calcid, spaceid) FROM stdin;';
+const query = 'COPY public.reviews (id, date, rating, content, space, author_id) FROM stdin;';
```

Comment:
I like how you break everything out into specific files with clear names, makes it very easy to follow.